easybuild/easyconfigs/s/Sphinx/Sphinx-2.4.3-GCCcore-8.3.0-Python-3.7.4.eb

```diff
@@ -0,0 +1,78 @@
+easyblock = 'PythonBundle'
+
+name = 'Sphinx'
+version = '2.4.3'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://www.sphinx-doc.org'
+description = """Sphinx is a tool that makes it easy to create intelligent and beautiful documentation.
+ It was originally created for the new Python documentation, and it has excellent facilities for the 
+ documentation of Python projects, but C/C++ is already supported as well, and it is planned to add 
+ special support for other languages as well."""
+
+toolchain = {'name': 'GCCcore', 'version': '8.3.0'}
+
+builddependencies = [('binutils', '2.32')]
+
+dependencies = [('Python', '3.7.4')]
+
+exts_default_options = {
+    'source_urls': [PYPI_SOURCE],
+    'sanity_pip_check': False,
```

Comment:
@furstj There's no need to set `sanity_pip_check` to `False` here (especially not since you're setting it to `True` below?

Please change to:

```python
exts_default_options = {'source_urls': [PYPI_SOURCE]}

Reply:
@boegel Unfortunately I deleted my branch with Sphinx :(. Please follow Sphinx in pr #10012

Reply:
No problem, following up in #10012...